src/lib/mdns/minimal/responders/Ptr.h

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <mdns/minimal/records/Ptr.h>
+#include <mdns/minimal/responders/Responder.h>
+
+namespace mdns {
+namespace Minimal {
+
+class PtrResponder : public Responder
+{
+public:
+    PtrResponder(const FullQName & qname, const FullQName & target) : Responder(QType::PTR, qname), mTarget(target) {}
+
+    void AddAllResponses(const chip::Inet::IPPacketInfo * source, ResponderDelegate * delegate) override
+    {
+        PtrResourceRecord record(GetQName(), mTarget);
+        record.SetTtl(4500);
```

Comment:
Why 4500?

Reply:
I think I copied it from somewhere (some replies from chromecast I think). This was before I had a default TTL in recods.
TTL still needs to be solved, however for now I just removed this. Will have to add sane ttl in a subsequent pass.